services/emailService.js

```diff
@@ -1,34 +1,29 @@
 'use strict';
 
-var http = require('../common/http.js');
+var http = require('../common/httpClient.js');
 var core = require('../common/core.js');
 var Q = require('q');
 
 module.exports = function () {
-    var getIdpLatestEmail = function (userEmail, subject) {
-        return getLatestEmail(core.getConfig().idpEmailMockService.url, userEmail, subject).then(function (response) {
-            // Workaround for issue in ISG Email Stub where dots are sometimes duplicated (due to that registration link gets broken)
-            // Bus to provide more details
-            return JSON.parse(response.body.replace(/\.\./g, '.'));
-        });
-    };
-
     var getIdamLatestEmail = function (userEmail, subject) {
```

Comment:
I think we should keep both functions. The reason for that is our (IDAM emails) are not HTML and because of that the JSON produced by ISG Stub is wrong (newlines in JSON are not escaped) and it fails to parse it otherwise. That's why replace newlines with BR's. IDP Emails on the other hand are already HTML so no further processing should be done.

eg. our email:

```
Hello John,

How are you?

This is unmanned email
```
JSON: { message: "
Hello John,

How are you?

This is unmanned email
"} <-- Real newlines breaks parser
IDP Email:

```
Hello John,<br />How are you?<br />This is unmanned email
```

JSON:
```
{ message: "Hello John,<br />How are you?<br />This is unmanned email" }
```